### DISCUSSION TITLE
How to export energy probe tabular data

### DISCUSSION THREAD

**Discussion by sy13520** [ansys/pymechanical#989] [Q&A] [answered]
Hello everyone, I encountered a problem in my mechanic script learning, about how to use Python to export the tabular data generated in the energy probe, because I want to output the energy size at the position where the slope of the strain energy curve is equal to 0, how should I export the tabular data in the energy probe to Excel to facilitate my data extraction? Thank you very much!

**Comment by MeenaBytes** [comment, marked as answer]
@sy13520  Please refer to couple of solutions [here](https://discuss.ansys.com/discussion/255/how-to-access-tabular-data-using-act). Unfortunately, it will not work in the batch mode.

**Reply by sy13520** [reply]
Thank you very much for your answer, but is there any code that can be used in Python environment?
My project may need to simulate thousands of models, but they are all repetitive operations, or can I extract the data of these two extreme values? Thank you very much for your answer
![20241203110315](https://github.com/user-attachments/assets/6db3af40-169a-450d-985b-2fb1fc6670ac)

**Reply by MeenaBytes** [reply]
Hi @sy13520 , The post is having solutions through Python. You can run those script via PyMechanical remote [session](https://mechanical.docs.pyansys.com/version/stable/user_guide_session/index.html). once you get data from mechanical session, you should be able to find maxima and minima.
